cfme/intelligence/reports/reports.py

```diff
@@ -302,7 +319,6 @@ def update(self, updates):
         else:
             view.cancel_button.click()
         view = self.create_view(ReportDetailsView, override=updates)
-        assert view.is_displayed
```

Comment:
@john-dupuy as you are removing `view.is_displayed` please add `wait` above while creating view.